pkg/virt-config/configuration_test.go

```diff
@@ -644,24 +644,4 @@ var _ = Describe("test configuration", func() {
 		Entry("ClusterProfiler feature gate enabled should result in cluster profiler being enabled",
 			[]string{virtconfig.ClusterProfiler}, true),
 	)
```

Comment:
why are we removing the `LiveMigration`, if the PR talks about only removing the `SRIOVLiveMigration` feature gate?
Perhaps we should keep the test with only `LiveMigration` feature gate check